eos-tech-support/eos-diagnostics

```diff
@@ -556,7 +556,7 @@ let diagnostics = [
     {
         title: 'Journal',
         hardwareInfo: true,
-        content: function() { return trySpawn('journalctl --no-pager'); },
+        content: function() { return trySpawn('journalctl --no-pager --output verbose'); },
```

Comment:
I don't think this is such a good idea.  I just checked on a system that I rebooted a few minutes ago, and this would bloat the diagnostic file size from 2688 lines / 277 kB to 46241 lines / 1.9 MB.  And, looking at the content, I personally find it much more difficult to read through the resulting file.